README.md

```diff
@@ -485,7 +494,11 @@ The converted DAG uses the `BashOperator` in Airflow.
 
 ### Known limitations
 
-Not all FS operations are currently idempotent. This will be fixed.
+Not all FS operations are currently idempotent. It's not a problem if prepare action is used in other tasks
+but might be a problem in certain situations. Fixing the operators to be idempotent requires more complex
```

Comment:
As a new o2a user I would be glad to know an example of a "certain situation" 🙂

Reply:
Duplicate: https://github.com/GoogleCloudPlatform/oozie-to-airflow/pull/345/files#r303486063